com.unity.cinemachine/Samples~/samples.json

```diff
@@ -0,0 +1,31 @@
+{
+  "SharedAssetDependencies": [
+    "Shared"
```

Comment:
Why have this?  Why not just mention Shared in the package-specific asset dependencies?  Right now, you can't have samples that don't depend on Shared (such as OldSamplesFromCM2)